src/site/markdown/server/install/guice-cassandra-rabbitmq-swift.md

```diff
@@ -1,11 +1,11 @@
-# Guice-Cassandra-Rabbitmq installation guide
+# Guice-Cassandra-Rabbitmq-Swift installation guide
 
 ## Building
 
 ### Requirements
 
  - Java 8 SDK
- - Docker ∕ ElasticSearch 2.4.6 and Cassandra 3.11.3
+ - Docker ∕ ElasticSearch 2.4.6, RabbitMQ Management 3.3.7, Swift ObjectStorage and Cassandra 3.11.3
```

Comment:
which version of Swift ?

Reply:
see https://github.com/jeantil/openstack-swift-keystone-docker , it should be Swift 2.15.1

Reply:
oh, cool!!! I was struggling with finding its version by doing docker inspect =.=. Thank you any way!